File: Source/Aladdin.ASN1.ISO.Java/src/aladdin/asn1/iso/pkix/ce/ExtKeyUsageSyntax.java
package aladdin.asn1.iso.pkix.ce;
import aladdin.asn1.*; 
import java.io.*;

// ExtKeyUsageSyntax ::= SEQUENCE OF OBJECT IDENTIFIER

public final class ExtKeyUsageSyntax extends Sequence<ObjectIdentifier>
{
    private static final long serialVersionUID = 3395090662622544339L;
    
	// конструктор при раскодировании
	public ExtKeyUsageSyntax(IEncodable encodable) throws IOException 
	{
		super(ObjectIdentifier.class, encodable); 
	}
	// конструктор при закодировании
	public ExtKeyUsageSyntax(ObjectIdentifier... values) 
	{
		super(ObjectIdentifier.class, values); 
	}
}

File: Source/Aladdin.ASN1.ISO/ASN1/ISO/PKIX/CE/ExtKeyUsageSyntax.cs
using System; 
using System.Runtime.Serialization;

// ExtKeyUsageSyntax ::= SEQUENCE OF OBJECT IDENTIFIER

namespace Aladdin.ASN1.ISO.PKIX.CE
{
	[Serializable]
	public class ExtKeyUsageSyntax : Sequence<ObjectIdentifier>
	{
		// конструктор при сериализации
        protected ExtKeyUsageSyntax(SerializationInfo info, StreamingContext context) : base(info, context) {}

		// конструктор при раскодировании
		public ExtKeyUsageSyntax(IEncodable encodable) : base(encodable) {}

		// конструктор при закодировании
		public ExtKeyUsageSyntax(params ObjectIdentifier[] values) : base(values) {}
	}
}
